extensions/wikia/ArticleComments/js/ArticleComments.js

```diff
@@ -123,7 +123,8 @@
 					id: commentId,
 					method: 'axEdit',
 					rs: 'ArticleCommentsAjax',
-					useskin: window.skin
+					useskin: window.skin,
+					cb: Date.now()
```

Comment:
@jsutterfield, try sending a request directly to one of the Apache nodes to get the response headers that the CDN layer processes. The `Cache-Control` you see on the client-side is actually `X-Pass-Cache-Control` generated on the backend.